addons/web/static/src/views/list/list_renderer.js

```diff
@@ -287,13 +287,16 @@ export class ListRenderer extends Component {
         return Object.values(list.fields)
             .filter(
                 (field) =>
+                    list.activeFields[field.name] &&
                     field.relatedPropertyField &&
-                    field.relatedPropertyField.fieldName === column.name &&
+                    field.relatedPropertyField.name === column.name &&
                     field.type !== "separator"
             )
             .map((propertyField) => {
+                const activeField = list.activeFields[propertyField.name];
                 return {
                     ...getPropertyFieldInfo(propertyField),
+                    relatedPropertyField: activeField.relatedPropertyField,
```

Comment:
getPropertyFieldInfo already return relatedPropertyField. It not the correct one ?

Reply:
`relatedPropertyField` from the active field is the same info as the `relatedPropertyField` from field definitions